backend/pkg/backend/backend.go

```diff
@@ -106,7 +106,7 @@ func Run() {
 		log.Fatalf("Failed to create runtime scan orchestrator: %v", err)
 	}
 
-	restServer, err := rest.CreateRESTServer(config.BackendRestPort, backend.dbHandler, orc, k8sClientset)
+	restServer, err := rest.CreateRESTServer(config.BackendRestPort, backend.dbHandler, orc, k8sClientset, config.DefaultScanParallelism)
```

Comment:
IMO it will be better to set the default during `CreateScanner` (called from `createRuntimeScanOrchestrator`-->`orchestrator.Create`) and set it on `Scanner`. 
So the default value should be set on runtime_scan/pkg/config and not on the general config.
When we get into `(s *Scanner) jobBatchManagement()`  
```
	numberOfWorkers := s.scanConfig.MaxScanParallelism
```
`numberOfWorkers` - should be set from `scanConfig` and if empty should be taken from `s.defaultScanParallelism`

It will remove all the propegation of the default value in less relevant points like `rest.CreateRESTServer`.

WDYT? Hope the comment was clear enough.

Reply:
@FrimIdan 
Thank you for your comment :)
I implemented again.

Reply:
@MasayaAoyama Thank you, it looks a lot better now, just added a small comment and we good to go I think.